commit 03da55716ca1e277a573c2edca333a339ddc218a
Author: Kefu Chai <k.chai@proxmox.com>
Date:   Fri Jul 29 22:35:19 2022 +0800

    mgr/MgrCap: include <boost/phoenix.hpp>
    
    instead of include `spirit/include/phoenix.hpp`, include
    `<boost/phoenix.hpp>` to silence the warning from boost, like:
    
    /opt/ceph/include/boost/spirit/include/phoenix.hpp:12:1: note: ‘#pragma
    message: This header is deprecated. Use <boost/phoenix.hpp> instead.’
       12 | BOOST_HEADER_DEPRECATED("<boost/phoenix.hpp>")
          | ^~~~~~~~~~~~~~~~~~~~~~~
    
    Signed-off-by: Kefu Chai <tchaikov@gmail.com>

diff --git a/src/mgr/MgrCap.cc b/src/mgr/MgrCap.cc
index cba758083c3..1563516a9be 100644
--- a/src/mgr/MgrCap.cc
+++ b/src/mgr/MgrCap.cc
@@ -12,14 +12,14 @@
  *
  */
 
+#include <boost/algorithm/string/predicate.hpp>
 #include <boost/config/warning_disable.hpp>
-#include <boost/spirit/include/qi_uint.hpp>
-#include <boost/spirit/include/qi.hpp>
-#include <boost/fusion/include/std_pair.hpp>
-#include <boost/spirit/include/phoenix.hpp>
 #include <boost/fusion/adapted/struct/adapt_struct.hpp>
 #include <boost/fusion/include/adapt_struct.hpp>
-#include <boost/algorithm/string/predicate.hpp>
+#include <boost/fusion/include/std_pair.hpp>
+#include <boost/phoenix.hpp>
+#include <boost/spirit/include/qi.hpp>
+#include <boost/spirit/include/qi_uint.hpp>
 
 #include "MgrCap.h"
 #include "include/stringify.h"
